03-task-manager/starter/db/connect.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require("mongoose")
+
+const connectDB = (url) => {
```

Comment:
You can remove the `url` parameter here if you want to reference `process.env.MONGODB_URI` here directly instead of passing it in from `app.js`.